integration-cli/docker_cli_start_test.go

```diff
@@ -130,24 +126,23 @@ func (s *DockerSuite) TestStartMultipleContainers(c *check.C) {
 
 	out, err := inspectField("parent", "State.Running")
 	c.Assert(err, check.IsNil)
-	if out != "false" {
-		c.Fatal("Container should be stopped")
-	}
+	// Container should be stopped
+	c.Assert(out, checker.Equals, "false")
 
 	// start all the three containers, container `child_first` start first which should be failed
 	// container 'parent' start second and then start container 'child_second'
 	out, _, err = dockerCmdWithError("start", "child_first", "parent", "child_second")
-	if !strings.Contains(out, "Cannot start container child_first") || err == nil {
-		c.Fatal("Expected error but got none")
-	}
+	// Expected error but got none
+	c.Assert(err, checker.NotNil)
+	// Expected error but got none
+	c.Assert(out, checker.Contains, "Cannot start container child_first")
```

Comment:
The comment could be better ; "output does not correspond to what was expected" or something like that :wink: .